packages/paste-core/components/anchor/__tests__/anchor.test.tsx

```diff
@@ -179,4 +180,41 @@ describe('Anchor', () => {
       expect(results).toHaveNoViolations();
     });
   });
+
+  describe('Customization', () => {
+    it('should set a element data attribute for Anchor', () => {
+      render(
+        <Anchor href="https://twilio.com">
+          This is an anchor that links to Twilio.com with an external target and rel
+        </Anchor>
+      );
+      const renderedAnchor = screen.getByRole('link');
+      expect(renderedAnchor.getAttribute('data-paste-element')).toEqual('ANCHOR');
+    });
+    it('should set a custom element data attribute for Anchor', () => {
+      render(
+        <Anchor element="FUNKY_ANCHOR" href="https://twilio.com">
+          This is an anchor that links to Twilio.com with an external target and rel
+        </Anchor>
+      );
+      const renderedAnchor = screen.getByRole('link');
+      expect(renderedAnchor.getAttribute('data-paste-element')).toEqual('FUNKY_ANCHOR');
+    });
+
+    it('should add custom styles to Anchor', (): void => {
```

Comment:
Would it be good to also include a test for adding custom styles to Anchor when a custom element data attribute is being used?

Reply:
Good call!